package/python-kiwi-spec-template

```diff
@@ -383,13 +381,16 @@ overlay root filesystem
 Summary:        KIWI - buildservice package requirements for boot images
 Provides:       kiwi-boot:netboot
 Provides:       kiwi-boot:oemboot
+Provides:       kiwi-image:docker
```

Comment:
Sorry that doesn't make sense to me. The kiwi-image:`image-type-name` provides indicates that this package provides the image type `image-type-name`. Support for the image types is provided by the python-kiwi base package aka python2-kiwi and python3-kiwi but not by the package kiwi-boot-requires where you would like to move it. Could you explain why this should be correct ?

Reply:
Even more confusing is that the docker type doesn't need a boot image at all which means for building docker images obs doesn't need kiwi-boot-requires at all. If the provides tag which indicates this kiwi can build docker images is now moved into a package which is not needed to build docker images we will completely loose the information. That can't be the intention, or did I misunderstood the concept ?

Reply:
it indicates that it provides all tools and therefore all it dependencies to be able to build that image.

Currently docker builds are failing due to missing tools and despite the fact that this provider is installed.